PLUB/Configuration/Manager/UserManager.swift

```diff
@@ -106,5 +111,30 @@ extension UserManager {
         }
       }
   }
+  
+  /// 최근 검색어목록에 키워드를 추가합니다
+  /// - Parameter keyword: 검색한 키워드
+  func addKeyword(keyword: String) {
+    let recentKeywordList = recentKeywordList ?? []
+    var filterList = recentKeywordList.filter { $0 != keyword }
+    filterList.insert(keyword, at: 0)
+    self.recentKeywordList = filterList
+  }
+  
+  /// 최근 검색어목록에 특정 인덱스에 위치한 키워드를 삭제합니다
+  /// - Parameter index: 삭제하고자하는 키워드의 위치
+  func removeKeyword(index: Int) {
+    guard var recentKeywordList = recentKeywordList,
+          index < recentKeywordList.count && 0 <= index else { return }
```

Comment:
패턴매칭을 이용해보셔도 좋을 것 같아요 :)
```suggestion
    guard var recentKeywordList = recentKeywordList,
          (0..<recentKeywordList.count) ~= index else { return }
```

Reply:
와 뭐야 이런건 또 처음 보네요 ㅋㅋㅋㅋㅋㅋ 수정하도록 하겠습니다 ☺️☺️